protocol/src/main/scala/io/mediachain/protocol/Datastore.scala

```diff
@@ -95,12 +101,6 @@ object Datastore {
       meta.get("type")
```

Comment:
Will this still work?  Now that `meta` is an explicit field, it shouldn't contain the type name... I added this so that we could deserialize a subtype to the base type but still preserve the original type name in case we re-serialize it later...